### DISCUSSION TITLE
why the logs are different ???

### DISCUSSION THREAD

**Discussion by sanu2002** [Cyfrin/foundry-full-course-cu#2918] [General]
```
wen i am running with forge test -vv then right amount is credited but wen i am running this command ```forge test --fork-url $rpc_url -vv```
then it is giving me extra value why , if that type things will happen then it will be very difficult for me to work with the test data ??
```

![p_1](https://github.com/user-attachments/assets/b80cf5a2-d3a7-4ff4-aaec-c990e1bf2807)
![p_2](https://github.com/user-attachments/assets/17137446-015c-404f-abad-846eb4ed6b67)

**Comment by EngrPips** [comment]
I am unsure what is the cause for this discrepancy. hmmmm 🤔

**Reply by sanu2002** [reply]
Same intended code but different logs 😕??

**Comment by sanu2002** [comment]
I got the solution , it's my mistake in conversion section i made the wrong calculation ...

**Reply by EngrPips** [reply]
Well, we are often the cause of our problems.
